packages/backend-for-frontend/.env

```diff
@@ -14,6 +14,7 @@ PURPOSE_PROCESS_URL="http://localhost:3400"
 AUTHORIZATION_PROCESS_URL="http://localhost:3300"
 DELEGATION_PROCESS_URL="http://localhost:3800"
 ESERVICE_TEMPLATE_PROCESS_URL="http://localhost:3900"
+NOTIFICATION_CONFIG_PROCESS_URL="http://localhost:4100"
```

Comment:
this env var and the feature flag env var should be added to all environments, so the config can be parsed correctly

I'll open a PR for this in the deployment repo

Reply:
https://github.com/pagopa/interop-core-deployment/pull/279